pkg/client/status_reporter.go

```diff
@@ -27,12 +27,24 @@ import (
 )
 
 const (
+	partiallyAvailable          string = "Rollout of the monitoring stack failed but is Available. Please investigate the degrades status error."
```

Comment:
@simonpasquier @jan--f does it make sense to have message like this when `Available` is true but is degraded? This message will be set on `Available` condition  when the system has `degraded` but is `available`